staramr/blast/plasmidfinder/PlasmidfinderBlastDatabase.py

```diff
@@ -12,12 +14,54 @@
 
 class PlasmidfinderBlastDatabase(AbstractBlastDatabase):
 
-    def __init__(self, database_dir: str) -> None:
+    def __init__(self, database_dir: str, bacteria=None) -> None:
         """
         Creates a new PlasmidfinderBlastDatabase.
-        :param database_dir: The specific ResFinder database (drug class) directory.
+        :param database_dir: The specific PlasmidFinder database directory.
         """
+
         super().__init__(database_dir)
+        self.plasmidfinder_database_dir = path.join(self.database_dir)
+        self.bacteria = bacteria
+
+        if bacteria is not None:
+            self.plasmidfinder_database_file = path.join(self.database_dir, bacteria)
+            self.plasmidfinder_database_file += ".fsa"
```

Comment:
Can you use `self.fasta_suffix` instead (which should be equal to `".fsa"`).

Reply:
Yeah, resolved this in my next commit